app/static/js/admin/auth/plus.js

```diff
@@ -39,25 +39,17 @@ var helper = (function() {
                 url: '/admin/store-token?state=' + STATE + '&next=' + NEXT,
                 contentType: 'application/octet-stream; charset=utf-8',
                 success: function(result) {
-                    window.location.href = result;
+                    window.location.href = result.data.redirect_url;
                 },
                 error: function(xhr, status, err) {
-                    if (xhr.status == 200) {
```

Comment:
I think this can be removed... it was a bug that was happening because we weren't returning valid JSON, just a string, so jQuery thought it was a bad request....